tests/rt.sh

```diff
@@ -413,7 +413,7 @@ if [[ $TESTS_FILE =~ '35d' ]]; then
   TEST_35D=true
 fi
 
-BL_DATE=20210401
+BL_DATE=develop-20210406
```

Comment:
Please fix this. The BL_DATE should not include the word "develop-".

Reply:
> Please fix this. The BL_DATE should not include the word "develop-".

@DeniseWorthen : Already fixed and will be committed after the rt done

Reply:
I cannot verify baselines on cheyenne until you push this change.